### DISCUSSION TITLE
[design/antagonist] Minor changeling rework.

### DISCUSSION THREAD

**Discussion by hal9000PR** [ParadiseSS13/Paradise#17866] [Documents]
## Goals

- Make changelings less reliant on a handful of fairly uninteresting abilities.
- Lean further into the theme of "masters of evolution" apex preditors

##  The Idea.
delete fleshmend, epi overdose, speedlegs, and the toxin heal abilities.

Add one new ability, all changelings get by default, "Master of evolution"

Using this ability would bring up a TGUI menu, with multiple sliders, all of these are linked to a related stat.
moving these sliders would increase or decrease a given stat, by default, you would be only able to increase one stat by a small amount, you can however, go into negatives with a different stat, to give you more points to move.

## STATS

SPEED:
 - increasing it will make you faster.
 - decreasing it will make you slower.
 
 BODILY FUNCTIONS:
 - increasing it will make you rapidly heal toxin damage, organ damage, rads, mutations, toxic chemicals, etc.
 - decreasing it will make you take toxin damage over time, decreasing it further will halt breathing.
 
 REGENERATION:
 - increasing it will give you pain resistance and healing of physical damages.
 - decreasing it will make you take brute damage and blood loss over time.
 
 TENACITY:
 - increasing it from 0 will cleanse stuns and stamina.
 - increasing this stat will have a reduction on stuns and stamina damage.
 - decreasing it will increase stamina damage taken and make stuns last longer.
 
 FORTITUDE:
 - increasing it will reduce physical damage taken.
 - decreasing it will increase physical damage taken.
 
 STRENGTH:
 - increasing it will decrease `do_after` times (aside for a few, might make an exception for applying handcuffs like TG does with their stuff like this), and increase melee damage.
 - decreasing it will increase `do_after` times, and decrease melee damage.
 
 if anyone can think of any other stats that they could get, I am open to suggestions.
   
## what stops a changeling from min maxing speed and stun resistance with memechem healing countering the damage?

`telomeres` are a long bundle of "useless" DNA, they basically sit at the top of your chromosomes like the end bit of your shoelace, and keep it protected.

If you overuse a gene (stat), it will run down your telomeres, after which, the gene will start taking "damage" which will reduce the maximum amount of "points" you can put into a given gene, and deal a small amount of clone damage, until you stop using the ability. at which point the gene will begin to recover. After the gene fully heals the telomeres will begin to heal.

**Comment by Inadvizable** [comment]
The ability to manipulate your stats on the fly is very on-brand for changelings, this is very interesting as a concept.
